nav2_bt_waypoint_follower/include/nav2_bt_waypoint_follower/bt_waypoint_follower.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 Samsung Research America
+// Copyright (c) 2020 ymd-stella
```

Comment:
... please don't remove my copyrights on my own files :heart:

Reply:
When I removed waypoint_follower and added bt_waypoint_follower, git seemed to make the determination that waypoint_follower.hpp was renamed to bt_waypoint_follower.hpp.
Since I'm not really interested in copywriting, can I write the following?
```
Copyright (c) 2020 Samsung Research America
```
How about that?

Reply:
You're welcome to add your own copyright line if you made substantive intellectual property changes to the original file you modified. So it could be both.

Reply:
I understand.
I'll fix it.

Reply:
I've written both.